ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -716,6 +716,12 @@ impl GlobalDecl {
         self.is_threadlocal
     }
 
+    /// Return whether the declaration is a thread local.
```

Comment:
Is this docstring right?

Reply:
Huh. I swear I fixed it. Must have gotten lost during my various rebases. Will fix.

Reply:
[ca9db08](https://github.com/ykjit/yk/pull/1664/commits/ca9db08f029e2607ce42ebd8a7e99078e391f6a7)